contributions-chapter/chapter.md

```diff
@@ -1,15 +1,98 @@
 
-This chapter outlines the conributions that each of the teams in the course were able to make to their open-source project. Contributions have been categorized and are presented below:
+This chapter outlines some of the contributions made by several teams to their open-source project. Contributions have been categorized based on their types. At the end a list of all mentioned Pull Requests will be provided for more information. It should be mentioned that much more Pull Requests were filed during the course, this chapter only sketches a broad overview of the diversity.
 
-## Language/Typo/Link fixes
-- This [pull request](https://github.com/elastic/kibana/pull/10709) fixes some language mistakes in the `FAQ.md` file of Kibana's documentation.
-- This [pull request](https://github.com/elastic/kibana/pull/10714) fixes a typo in the `kbn_server.js` file in the Kibana project.
-- This [pull request](https://github.com/elastic/kibana/pull/10715) fixes a broken link in Kibana's `CONTRIBUTING.md` file.
+## Documentation fixes
+
+Several teams proposed changes to the documentation of their projects. This could go as far as fixing typo's or updating the documentation for deprecated / added features.
+One of the teams to file such Pull Requests was Kibana ([#10709][KI10709] :white_check_mark:, [#10714][KI10714] :white_check_mark:, [#10715][KI10715] :white_check_mark:). Often, these Pull Requests were merged without much discussion.
+
+[KI10709]: https://github.com/elastic/kibana/pull/10709
+[KI10714]: https://github.com/elastic/kibana/pull/10714
+[KI10715]: https://github.com/elastic/kibana/pull/10715
 
 ## Bug fixes
-- This Kibana [issue](https://github.com/elastic/kibana/issues/9108) depicts a problem in Safari where exporting data would lead to a display of the JSON in the browser instead of a download action (labeled P3: nice to have bug-fixes). Investigation led to finding out it was a [bug](https://bugs.webkit.org/show_bug.cgi?id=102914) in Webkit that has recently been fixed. An upgraded version of Safari (10.1) should thus solve the problem. Workaround possibilities exist, but these are unreliable. Hence, this analysis was posted in the [issue](https://github.com/elastic/kibana/issues/9108) on Github along with a suggesion for the Kibana team to wait for the new version of Safari. For more info on the download library please visit this [link](eligrey/FileSaver.js#12).
+
+Several teams went through the issue lists of their projects to find bugs to fix.
+An example would be the [bug][VS-bug] in the search functionality of Visual Studio Code:
+when a user searched for two terms in Visual Studio Code, the 'clear results' button was clicked  and alt+up was used to get the last search item, the second last was shown.
+
+To get the last history item the function `showPreviousSearchTerm()` was called in the file `src\vs\workbench\parts\search\browser\searchWidget.ts`. This function is shown in [Snippet 1](#snippet-1)
+
+<div id="snippet-1"></div>
```

Comment:
There should be a new line after each closing tag. It breaks the snippet rendering